enclave/CMakeLists.txt

```diff
@@ -56,14 +48,18 @@ if (CONSENSUS)
     list(REMOVE_ITEM CPU_SOURCES ${PROJECT_SOURCE_DIR}/src/c_api/c_api.cc)
     add_custom_command(OUTPUT xgboost_mc_t.h xgboost_mc_t.c xgboost_mc_args.h 
         DEPENDS ${PROJECT_SOURCE_DIR}/xgboost_mc.edl
-        COMMAND openenclave::oeedger8r --trusted ${PROJECT_SOURCE_DIR}/xgboost_mc.edl)
+        COMMAND openenclave::oeedger8r --trusted ${PROJECT_SOURCE_DIR}/xgboost_mc.edl
+        --search-path ${OE_INCLUDEDIR} --search-path
+        ${OE_INCLUDEDIR}/openenclave/edl/sgx)
```

Comment:
Does this make Secure XGBoost compatible with only SGX?

Reply:
No, this just updates the path to include some SGX-specific OE .edl files (but doesn't exclude others)